test/functional_ccs/apps/discover/saved_queries_ccs.ts

```diff
@@ -140,8 +140,9 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
         expect(await filterBar.hasFilter('extension.raw', 'jpg')).to.be(true);
         expect(timePickerValues.start).to.not.eql(PageObjects.timePicker.defaultStartTime);
         expect(timePickerValues.end).to.not.eql(PageObjects.timePicker.defaultEndTime);
-        await retry.waitFor(
+        await retry.waitForWithTimeout(
```

Comment:
Same comment here on logging the actual count.  Either every retry, or at least once at the end.